packages/core/test/iframe.ts

```diff
@@ -7,7 +7,10 @@ import {
     HashParamsRetriever,
 } from './test-api-service';
 
-const com = new Communication(window, self.name);
+const options = new URLSearchParams(window.location.search);
+const instanceId = options.get(INSTANCE_ID_PARAM_NAME)!;
+const com = new Communication(window, self.name || instanceId);
```

Comment:
Why do we use `self.name` if instanceId is always provided? And why do we need to set the iframe name at all?